rdrf/registry/patients/admin_forms.py

```diff
@@ -564,12 +564,22 @@ def clean_registered_clinicians(self):
         return reg_clinicians
 
     def clean_email(self):
+        registries = self.cleaned_data.get("rdrf_registry", Registry.objects.none())
+        email = self.cleaned_data.get("email")
+
+        if "email" in self.changed_data:
+            for registry in registries:
+                if registry.has_feature(RegistryFeatures.PATIENTS_CREATE_USERS):
+                    if CustomUser.objects.filter(email=email).exists():
```

Comment:
Should probably do a case insensitive check here i.e. `test@email.com` is the same email as `TEST@email.com`